src/index.json

```
@@ -3146,4 +3146,4 @@
         ]
     },
     "formatVersion": "1"
-}
\ No newline at end of file
+}
```

```suggestion
}
```
Remove the newline at the end of file. It's added by Github.